README.md

```diff
@@ -57,9 +56,6 @@ jump_ops = [np.sqrt(kappa) * a]
 rho0 = qt.coherent_dm(n, alpha0)
 tsave = np.linspace(0, 1.0, 101)
 
-# uncomment the next line to run the simulation on GPU
-# torch.set_default_device('gpu')
-
```

Comment:
> I also removed the "uncomment for GPU" lines since we'll soon have a dedicated tutorial, and I wasn't sure it was working (at least it was not being checked by doctest).

I think we should put back this line here and in the next example. The main goal of these two examples is to showcase the main features of the library in the smallest amount of code. It's important that people who don't know GPUs or PyTorch see that they just need one extra line and everything works the same.

Reply:
But does it work? We don't need to specify `device='cuda'` in the `options`? I didn't try but I'm quite skeptical this won't raise an error.

Reply:
I added it back on the first example. Note that the GPU example was not working, because we had `torch.set_default_device('gpu')` instead of `torch.set_default_device('cuda')`.

Reply:
Thanks! (related to https://linear.app/dynamiqs/issue/DYN-107/add-a-gpu-runner-to-the-ci for this kind of testing)

Reply:
I'd say we should add back the line to both examples (bc it shows that you can also do the gradient computation on GPU, as simply as running the simulation).